Upgrade dcm2niix to latest.

kousu opened the pull request:
Fixes #42, and fixes the broken tests in #18 so it will need to be rebased under #18 for that one to work.

jcohenadad commented:
@evaalonsoortiz there are 68 files changed, which makes it difficult for me to review. 

Many of the changes (if not all) are related to a move of files. If you wish to move files, please use `git mv`, so that these files are not considered "new" in the tracking (and therefore the review is less time consuming).

To make the review process easier, I would suggest you close this PR and reopen a new one based on a branch where files are moved using the command `git mv`. Also please, summarize the proposed changes for the PR in the description box (e.g., what are the main functionalities introduced, etc.). Thanks

rtopfer commented:
@evaalonsoortiz @jcohenadad 
head's up: i have a fairly significant redesign in the works (issue #41 ) for the object model (i.e. the class structure, which all eminates from MaRdI.m)

we can discuss it during Tuesday's meeting, but if you can avoid unnecessary changes (particularly to MaRdI.m) in the next few days that will make the potential merge much easier!

jcohenadad commented:
> @evaalonsoortiz @jcohenadad
> head's up: i have a fairly significant redesign in the works (issue #41 ) for the object model (i.e. the class structure, which all eminates from MaRdI.m)
> 
> we can discuss it during Tuesday's meeting, but if you can avoid unnecessary changes (particularly to MaRdI.m) in the next few days that will make the potential merge much easier!

@rtopfer sounds good. Could you please already open a PR from your branch (with prefix "(WIP)" in the title) so we are aware of the current developments. Thanks!

evaalonsoortiz commented:
This PR is being closed because the changes made are already in master.

kousu committed:
Upgrade dcm2niix to latest.

dcm2bids has only ever been tested with the latest version,
and dcm2niix is still shaking out its API bugs, so the only
reliable thing to do is to demand everyone stay up to date.

This is okay almost everywhere except on Debian which is always
slow to upgrade stuff.

jcohenadad committed:
Merge branch 'master' into ko/27-dcm2niix-ci
